frontend/src/index.css

```diff
@@ -1,9 +1,14 @@
  /* SPDX-License-Identifier: MIT */
  /* SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de> */
 
- @import url('http://fonts.googleapis.com/css?family=Noto+Sans:400,700,400italic,700italic&subset=latin,latin-ext');
- @import url('http://fonts.googleapis.com/css?family=Josefin+Sans:400,700,400italic,700italic&subset=latin,latin-ext');
- @import url('http://fonts.googleapis.com/css?family=PT+Sans:400,700,400italic,700italic&subset=latin,latin-ext');
+@import url('http://fonts.googleapis.com/css?family=Noto+Sans:400,700,400italic,700italic&subset=latin,latin-ext');
+@import url('http://fonts.googleapis.com/css?family=Josefin+Sans:400,700,400italic,700italic&subset=latin,latin-ext');
+@import url('http://fonts.googleapis.com/css?family=PT+Sans:400,700,400italic,700italic&subset=latin,latin-ext');
```

Comment:
What is the change here?

Reply:
The mockups specified some non default fonts, which are imported here.